sdc/tests/test_io.py

```diff
@@ -72,8 +72,30 @@ def setUp(self):
             A = np.random.ranf(n)
             A.tofile("np_file1.dat")
 
-    @unittest.skip('Error - fix needed\n'
-                   'NUMA_PES=3 build')
+    def assertDataEqual(self, left, right):
+        if isinstance(right, pd.Series):
```

Comment:
Why not to check also type of `left` value?

Reply:
`right` is expected value. It is used only for chosing comparison algorithm.